File: Daily Challanges/2026-04-April/2026-04-05.js
function getRotation(n) {
  let s = String(n);
  const k = s.length;
  for (let i = 0; i < k; i++) {
    const rotated = Number(s);
    if (rotated % k === 0) {
      return i;
    }
    s = s.slice(1) + s[0];
  }
  return "none";
}

File: Daily Challanges/2026-04-April/2026-04-05.py
def get_rotation(n):
    s = str(n)
    k = len(s)
    for i in range(k):
        rotated = int(s)
        if rotated % k == 0:
            return i
        s = s[1:] + s[0]
    return "none"


print(get_rotation(123))
